doc/nrf/ug_thread_tools.rst

```diff
@@ -0,0 +1,84 @@
+.. _ug_thread_tools:
+
+Thread tools
+************
+
+When working with Thread in |NCS|, you can use the following tools during Thread application development:
+
+* `nRF Thread Topology Monitor`_ - This desktop application helps to visualize the current network topology.
+* `nRF Sniffer for 802.15.4 based on nRF52840 with Wireshark`_ - Tool for analyzing network traffic during development.
+* :ref:`ug_thread_tools_wpantund`
+* `PySpinel`_ - Tool for controlling OpenThread Network Co-Processor instances through command line interface.
+
+Using Thread tools is optional.
+
+.. _ug_thread_tools_wpantund:
+
+wpantund
+========
+
+`wpantund`_ is a utility for providing a native IPv6 interface to a Network Co-Processor.
+When working with Thread, it is used for the interaction with the application by the following samples:
+
+* :ref:`ot_ncp_sample`
+
+The interaction is possible using commands proper of wpanctl, a module installed with wpantund.
```

Comment:
Did you mention writing "commands prompt of wpanctl" or "proper to wpanctl"? "Commands proper of" doesn't seems to be right to me.

Reply:
Good catch. Edited.